content/concepts/transport.md

```diff
@@ -115,3 +115,190 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+Transport protocols continue to improve transport methods in an attempt to alleviate 
+the shortcomings of the transport layer. 
+
+{{% notice "note" %}}
+
+Recalling the purpose of transports
+
+The IP service model provides logical communication between hosts (or nodes) but 
+is considered a best-effort delivery service, as segment delivery is not guaranteed. 
+The primary responsibility of transport protocols is to extend the IP 
+delivery service between two end systems. TCP connects the unreliable service of IP 
+between end systems into a reliable transport service between processes (i.e., the 
+processes running on the end systems). The purpose of newer transport protocols is not 
+only to improve current transport methods but also to allow for efficient connections 
+that distributed and peer-to-peer network stacks can utilize, like libp2p.
+
+{{% /notice %}}
+
+We need a transport protocol that:
+
+- Understands streams 
+- Is not byte-ordered
+- Overcomes HOL blocking (Head-of-line blocking)
+- Overcomes the latency of connection setup
+- Overcomes the ossification risks of TCP
+
+and, ideally, keeps the guarantees of TCP.
+
+In 2014, a new transport protocol 
+called QUIC (which, at the time, stood for Quick UDP Internet Connections, but now 
```

Comment:
Please don’t use the abbreviation any more, it would be nice if people forgot about this ;)

Reply:
Gotcha:)